applications/nrf_desktop/configuration/nrf52820dongle_nrf52820/prj.conf

```diff
@@ -36,10 +36,12 @@ CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE=y
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1536
 CONFIG_ISR_STACK_SIZE=1280
 CONFIG_MAIN_STACK_SIZE=840
-CONFIG_BT_RX_STACK_SIZE=2048
 CONFIG_BT_HCI_TX_STACK_SIZE_WITH_PROMPT=y
 CONFIG_BT_HCI_TX_STACK_SIZE=1536
 
+# Reuse system workqueue as Bluetooth RX context to reduce memory consumption
+CONFIG_BT_RECV_WORKQ_SYS=y
```

Comment:
If we configured the stack size for the BT RX thread like this:

`CONFIG_BT_RX_STACK_SIZE=2048` 

Shouldn't we make the stack size for the System Workqueue to be of equal size at least to the previous configuration?

`CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048`

Reply:
You are right that increasing system workqueue stack size would ensure that we are on safe side here. Firmware still fits after the increase.

btw. At some point we could think about tracking actual stack usages of our devices to prevent surprizes.